packages/pcd/semaphore-identity-v4/src/utils.ts

```diff
@@ -0,0 +1,54 @@
+import { decodePublicKey, encodePrivateKey, encodePublicKey } from "@pcd/pod";
```

Comment:
Not important in this PR, but this makes me think maybe it's time to pull some parts of podUtil and podCrypto into a separate package (crypto-util?) for reusability here.  This PCD doesn't really depend on PODs, just EdDSA signatures.